QUICKSTART.rst

```diff
@@ -136,10 +136,10 @@ What's next?
     changes since the last release              ``CHANGES.md``
 =============================================== =========================
 
-.. _platforms: http://chapel.cray.com/docs/1.13/platforms/index.html
-.. _chapel.cray.com/docs: http://chapel.cray.com/docs/1.13/
-.. _doc/building.rst: http://chapel.cray.com/docs/1.13/usingchapel/building.html
-.. _doc/compiling.rst: http://chapel.cray.com/docs/1.13/usingchapel/compiling.html
-.. _doc/executing.rst: http://chapel.cray.com/docs/1.13/usingchapel/executing.html
-.. _doc/debugging.rst: http://chapel.cray.com/docs/1.13/usingchapel/debugging.html
-.. _doc/bugs.rst: http://chapel.cray.com/docs/1.13/usingchapel/bugs.html
+.. _platforms: http://chapel.cray.com/docs/1.14/platforms/index.html
```

Comment:
These will conflict with #4582, but I've requested that we drop the version numbers entirely from these cross-references. This should PR should maybe wait until #4582 is merged.